Source/MSBuild.Sdk.Extras/Build/FrameworkResolvers/MSBuild.NETPortable.v4.5.targets

```diff
@@ -1,51 +1,51 @@
-﻿<Project>
-  <PropertyGroup>
-    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
-  </PropertyGroup>
-
+﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <PropertyGroup Condition="$(_SdkShortFrameworkProfileCount) == 2 and
-      $(_SdkShortFrameworkProfile.Contains('+net45+')) and
-      ($(_SdkShortFrameworkProfile.Contains('+win+')) or $(_SdkShortFrameworkProfile.Contains('+win8+')) or $(_SdkShortFrameworkProfile.Contains('+win80+')))">
+  <PropertyGroup Condition="$(_ShortFrameworkProfileCount) == 2 AND
+      $(_ShortFrameworkProfile.Contains('+net45+')) AND
+      ($(_ShortFrameworkProfile.Contains('+win+')) OR $(_ShortFrameworkProfile.Contains('+win8+')) OR $(_ShortFrameworkProfile.Contains('+win80+')))">
```

Comment:
Please do not rename all of the internal variables like this. I explicitly want them prefixed as _Sdk* so I don't accidently step-on others.

Reply:
They don't have any equivalent in the Main SDK. Also I have tested these properties, for a long time, none of the internal properties clash with the Main SDK. Some properties do blend with the Main SDK without affecting the build.

Reply:
It was easy for me to debug the targets since the naming conventions are similar with Main SDK.